.github/workflows/nodejs.yml

```diff
@@ -3,11 +3,7 @@
 
 name: Node.js CI
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: [push, pull_request, workflow_dispatch]
```

Comment:
I think on `push` we still want the branch param, but not for `pull_request` and `workflow_dispatch`. If it's on `push`, each test will run twice for every push to a branch with a PR.